terraform/aws/main.tf

```diff
@@ -78,8 +78,16 @@ resource "aws_subnet" "public" {
   }
 }
 
+locals {
```

Comment:
We have already devnet name in our CLI, so let's just pass it:
- https://github.com/dashevo/dash-network-deploy/blob/3f23a9f400ece89c4aee996a231f9f98a2cfb836/lib/cli/init.sh#L7
- https://github.com/dashevo/dash-network-deploy/blob/3f23a9f400ece89c4aee996a231f9f98a2cfb836/lib/cli/terraform.sh#L49

Reply:
This hack is to erase "devnet-" from network name

Reply:
Yes, but you can use env variable from cli

Reply:
with cli we are still used prefixed name, so we can rely on using this variable.

Reply:
`NETWORK_DEVNET_NAME` is what you need.

Reply:
done